dependencies/FindO2GPU.cmake

```diff
@@ -149,7 +149,7 @@ if(ENABLE_HIP)
     endif()
   endif()
 
-  if(HIP_PATH AND EXISTS "${HIP_PATH}" AND HCC_HOME AND EXISTS "${HCC_HOME}")
+  if(EXISTS "${HIP_PATH}" AND EXISTS "${HCC_HOME}")
     get_filename_component(hip_ROOT "${HIP_PATH}" ABSOLUTE)
     get_filename_component(hcc_ROOT "${HCC_HOME}" ABSOLUTE)
     find_package(hip)
```

Comment:
Ciao @davidrohr,
please add the following here:
`find_package(rocprim REQUIRED)`
This fixes the compilation on our workstation.